software/lamp-os/src/util/color.cpp

```diff
@@ -40,4 +40,18 @@ bool Color::operator==(const Color &inColor) const {
       b == inColor.b &&
       w == inColor.w);
 };
-}  // namespace lamp
\ No newline at end of file
+
+Color Color::lerp(const Color& other, float t) const {
```

Comment:
I think this will make the color object too big as a class member. Since we probably want to get rid of floats anyway, this could probably be expressed as a lerp/fade(startColor, endColor, percentage) and use the linear easing function already mapped in fade.cpp